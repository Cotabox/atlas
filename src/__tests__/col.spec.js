import { shallowMount } from '@vue/test-utils';
import Col from '../col.vue';

describe('Col', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Col);
  });

  it('should match with the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
