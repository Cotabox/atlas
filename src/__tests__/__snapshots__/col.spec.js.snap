// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col should match with the snapshot 1`] = `<div class="a-col-24 a-col-offset-0" style="padding-left: 0px; padding-right: 0px;"></div>`;
